Readme.md

```diff
@@ -1,76 +1,169 @@
+3. This will automatically trigger the `update-staging-version.yml` workflow inside your repository. 
+
+This process sends the new tag to the [graasp-deploy](https://github.com/graasp/graasp-deploy) repository and creates a new `YYYYMMddhhmm-staging-versions.json` file including the new version. 
+
+For more information on the appropriate tag syntax, see [???]. 
+
+### Deploy a new stack to staging environment
+The Continuous Delivery workflow (`cdelivery-trigger.yml`) is responsible of deploying a new stack of all the ecosystem versions to staging environment. It can be manually run from the Actions tab of the 
+[graasp-deploy](https://github.com/graasp/graasp-deploy) repository. Being a member of the **staging** team is required to perform these steps.
 
 ![Action tab](assets/images/learn-github-actions-repository.png)
 
-Write access to the repository is required to perform these steps.
+Whenever you want to deploy **the latest** staging stack available inside the [staging-versions](https://github.com/graasp/graasp-deploy/staging-versions) folder, you should perform the following steps:
 
-1. In the left sidebar, click the workflow you want to run.
+1. In the left sidebar, click the **Deploy to staging environment** workflow.
 
-    ![Action select](assets/images/actions-select-workflow.png)
+    ![Action select staging workflow](assets/images/actions-select-deploy-staging-workflow.png)
 
 2. Above the list of workflow runs, select Run workflow.
 
     ![Action dispatch](assets/images/actions-workflow-dispatch.png)
 
-3. Use the Branch dropdown to select the workflow's branch. Click Run workflow.
+3. (Optional) Use the input text field to specify a `YYYYMMddhhmm-staging-versions.json` file in case you want to deploy an specific stack. This file must be available inside the [staging-versions](https://github.com/graasp/graasp-deploy/staging-versions) folder. 
+
+    ![Workflow dropdown](assets/images/actions-dropdown-select-staging-stack.png)
 
-    ![Action run](assets/images/actions-manually-run-workflow.png)
+4. Finally, click on **Run workflow** and wait until the process is finished. 
 
-### Viewing a workflow
+This process automatically triggers all the `cdelivery-*.yml` workflows present in the repositories specified inside the `YYYYMMddhhmm-staging-versions.json` file. 
+After the workflow finishes the stack (represented by the `YYYYMMddhhmm-staging-versions.json` file) will be promoted to release:
+- It will be included in the `release-versions` folder
+- It will be copied to the `deployed/current-staging-versions.json` file. 
+
+
+### Deploy a new stack to production environment
+The Continuous Deployment workflow (`cdeployment-trigger.yml`) is responsible of deploying a new stack of all the ecosystem versions to production environment. It can be manually run from the Actions tab of the 
+[graasp-deploy](https://github.com/graasp/graasp-deploy) repository. Being a member of the **production** team is required to perform these steps.
```

Comment:
Does that mean you created those teams? How did you set the permissions for the teams?